app/javascript/.storybook/storybook.scss

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
This sets the stage for components displayed in Storybook. They will be centered vertically and horizontally in the Storybook work area.